bigtable/writes/write_simple.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package writes
+
+// [START bigtable_writes_simple]
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/bigtable"
+)
+
+func writeSimple(w io.Writer, projectID, instanceID string, tableName string) error {
```

Comment:
I generally try to avoid words like "simple" since this isn't simple to someone who doesn't know BigTable.

Reply:
It's simple because it is the most basic write we have for cloud bigtable, just writing data to one row.